### PR TITLE
MEN-5451: test: add acceptance test for the AWS IoT Core sync-devices CLI command

### PR DESCRIPTION
**Pull request by tranchitella**
Depends on: https://github.com/mendersoftware/integration/pull/2120
Includes: https://github.com/mendersoftware/iot-manager/pull/67

### CODE REVIEW

**Comment by tranchitella** [comment]
Depends on: https://github.com/mendersoftware/integration/pull/2120

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr integration/2120 --fast

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr integration/2120 --fast

File: tests/tests/test_sync_aws_iotcore.py [hunk 1]
```diff
@@ -0,0 +1,1371 @@
...
+                {
+                    "_id": dev["id"],
+                    "tenant_id": tenant_id,
+                    "integration_ids": [integration_id],
+                }
+                for dev in devices
+            ]
+            mgo.iot_manager.devices.insert_many(dev_docs)
+
+            # ... to be continued (FIXME)
```
Comment (alfrunes) [inline tests/tests/test_sync_aws_iotcore.py:1342]: This comment looks like it shouldn't be there :)
```suggestion
```

**Review by alfrunes — APPROVED** [review]

**Pull request merged**